### PR TITLE
Remove old version of JanDeDobbeleer.OhMyPosh

### PR DESCRIPTION
**Pull request by vedantmgoyal9**
null

### CODE REVIEW

**Comment by vedantmgoyal2009** [comment]
Any idea why is it failing?
image

File: manifests/s/SuperTux/SuperTux/0.6.2/SuperTux.SuperTux.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: SuperTux.SuperTux
+PackageVersion: 0.6.2
+PackageLocale: en-US
+Publisher: The SuperTux Development Team
```
Comment (ItzLevvie) [inline manifests/s/SuperTux/SuperTux/0.6.2/SuperTux.SuperTux.locale.en-US.yaml:6]:
```suggestion
Publisher: SuperTux Devel Team
```

File: manifests/s/SuperTux/SuperTux/0.6.2/SuperTux.SuperTux.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+PackageVersion: 0.6.2
+MinimumOSVersion: 10.0.0.0
+Installers:
+- InstallerLocale: en-US
+  Architecture: x64
+  InstallerType: msi
+  Scope: machine
+  InstallerUrl: https://github.com/SuperTux/supertux/releases/download/v0.6.2/SuperTux-v0.6.2-win64.msi
+  InstallerSha256: 4F858952AB26803657CD02607165668A37105054905012A6B95D7ACA59F4DEC3
+#  ProductCode:
```
Comment (ItzLevvie) [inline manifests/s/SuperTux/SuperTux/0.6.2/SuperTux.SuperTux.installer.yaml:13]:
```suggestion
  ProductCode: "{2CB8605B-F3FF-4AB7-BACB-2DDFB1B08D22}"
```

File: manifests/s/SuperTux/SuperTux/0.6.2/SuperTux.SuperTux.installer.yaml [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+  InstallerSha256: 4F858952AB26803657CD02607165668A37105054905012A6B95D7ACA59F4DEC3
+#  ProductCode: 
+  UpgradeBehavior: install
+- InstallerLocale: en-US
+  Architecture: x86
+  InstallerType: msi
+  Scope: machine
+  InstallerUrl: https://github.com/SuperTux/supertux/releases/download/v0.6.2/SuperTux-v0.6.2-win32.msi
+  InstallerSha256: 49E4D7361D75029B8FAD3F337534A5570B4C21F3270B69C35351C32654EB5C18
+#  ProductCode:
```
Comment (ItzLevvie) [inline manifests/s/SuperTux/SuperTux/0.6.2/SuperTux.SuperTux.installer.yaml:21]:
```suggestion
  ProductCode: "{39AE0518-A033-4491-94A5-5D3E3377E7B8}"
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Comment by ryfu-msft** [comment]
Hey @vedantmgoyal2009,

When performing a manual validation on an x86 host machine, I am able to install successfully, but I have issues running the executable. When I launch the executable, a blank window appears, then closes out immediately. Please verify your x86 installer and make sure that it can launch the installed app successfully on an x86 machine.

**Comment by vedantmgoyal2009** [comment]
Hey, can you please check again. On my machine nothing appears.

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
https://github.com/microsoft/winget-pkgs/pull/21665#issuecomment-882831658

Did you install the 32-bit application on a 32-bit OS?

Installing 32-bit applications on a 32-bit OS provides different results compared to installing 32-bit applications on a 64-bit OS.

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**